.github/workflows/cleanup-staging.yml

```diff
@@ -0,0 +1,43 @@
+name: Upload to staging
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@0.9.0
+        id: comment
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const isValidPreview = (
+              context.payload.issue.pull_request &&
+              context.payload.comment.body === '/preview'
+            );
+            return isValidPreview;
```

Comment:
```suggestion
```

We probably don't need this right?

Reply:
Sorry, deleted this when you were reviewing